docs/resources/rest_managed.md

```diff
@@ -0,0 +1,115 @@
+---
+# Documentation generated by "gen/generator.go"; DO NOT EDIT.
+# Copied from legacy-docs, in order to make changes change this file in legacy-docs and execute `go generate` from the repository root.
+# More details can be found in the [README](https://github.com/CiscoDevNet/terraform-provider-aci/blob/master/README.md).
+layout: "aci"
+page_title: "ACI: aci_rest_managed"
+sidebar_current: "docs-aci-resource-aci_rest_managed"
+subcategory: "Generic"
+description: |-
+  Manages ACI Model Objects via REST API calls. This resource can only manage a single API object and its direct children. It is able to read the state and therefore reconcile configuration drift.
+---
+
+# aci_rest_managed #
+
+Manages ACI Model Objects via REST API calls. This resource can only manage a single API object and its direct children. It is able to read the state and therefore reconcile configuration drift.
+
+## Example Usage ##
+
+The configuration snippet below creates a Tenant object.
+
+```terraform
+resource "aci_rest_managed" "example_tenant" {
+  dn         = "uni/tn-EXAMPLE_TENANT"
+  class_name = "fvTenant"
+  content = {
+    name  = "EXAMPLE_TENANT"
+    descr = "Example description"
+  }
+}
+```
+
+The configuration snippet below creates a Management Connectivity Preference object with an annotation overwrite.
+
+```terraform
+resource "aci_rest_managed" "example_management_connectivity" {
+  dn         = "uni/fabric/connectivityPrefs"
+  class_name = "mgmtConnectivityPrefs"
+  annotation = "orchestrator:example"
+  content = {
+    interfacePref = "ooband"
+  }
+}
+```
+
+The configuration snippet below creates a Tenant object with a VRF as child with annotation overwrites.
+
+```terraform
+resource "aci_rest_managed" "example_tenant_with_child" {
```

Comment:
Shall we add another direct child to show that you can add more than one?

Reply:
discussed not needed